src/components/powerstrip.js

```diff
@@ -33,11 +33,15 @@ class MiniMediaPlayerPowerstrip extends LitElement {
   }
 
   get sourceSize() {
-    return (this.config.source === 'icon' || this.config.collapse || this.idle);
+    return (this.config.source === 'icon' || this.config.hide.controls !== this.config.hide.volume || this.idle);
   }
 
   get soundSize() {
-    return (this.config.sound_mode === 'icon' || this.config.collapse || this.idle);
+    return (this.config.sound_mode === 'icon' || this.config.hide.controls !== this.config.hide.volume || this.idle);
+  }
+
+  get hasControls() {
+    return (this.config.collapse && this.player.active && !this.config.hide.controls);
```

Comment:
<mmp-media-controls> contains both volume & playback controls, so we need to check both `this.config.hide.controls` & `this.config.hide.volume` here.

Reply:
Fixed, had to reformat because of max line length.